content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -86,12 +86,36 @@ The output is similar to this:
 
 ### Without kubectl proxy
 
-Use `kubectl describe secret...` to get the token for the default service account with grep/cut:
+Use `kubectl apply` and `kubectl describe secret...` to create a token for the default service account with grep/cut:
+
+First, create the secret, requesting a token for the default service account:
```

Comment:
```suggestion
First, create the Secret, requesting a token for the default service account:
```
or (not really needed):
```suggestion
First, create the Secret, requesting a token for the default ServiceAccount:
```


We write Secret when we mean the Kubernetes kind; this helps readers realize that when we write “secreŧ” all-lowercase we might just mean it as a synonym for “confidential”.